### ISSUE TITLE
Cop.registry is deprecated

### ISSUE THREAD

**Issue by baukevw** [arthurnn/code-scanning-rubocop#17]
I'm getting this warning:

```code-scanning-rubocop-0.6.1/lib/code_scanning/rubocop/rule.rb:10: warning: `Cop.registry` is deprecated. Use `Registry.global` instead.```

Could this be addressed?

**Comment by baukevw** [comment]
@arthurnn can you review the PR I've created with the fix (#18)  and merge it and release the update?
